charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -83,6 +83,32 @@ spec:
         - mountPath: /var/lib/machine-controller-manager
           name: machine-controller-manager
           readOnly: true
+      - name: provider-openstack
```

Comment:
Please add this flag for explicit migration from [here](https://github.com/gardener/gardener-extension-provider-aws/pull/212/files#diff-e38adde4c814c39f0c3443b395a76dfc43639b003e8514c731b39e2b1683aea9R77), as this would take care of cleaning up of `OpenstackMachineClass` in case of migration triggered from MCM end.